packages/cli/src/services/util.ts

```diff
@@ -320,7 +320,10 @@ export async function loadPlaywrightProjectFiles (
     prefix,
   })
   for (const includePattern of include) {
-    archive.glob(includePattern, { cwd: dir }, {
+    archive.glob(includePattern, {
+      cwd: dir,
+      ignore: ignoredFiles,
```

Comment:
This change makes it so that our default ignore patterns (`['**/node_modules/**', '.git/**']`) now also apply to included files. I suppose that's OK though. I haven't tried if it works, but there's a chance that you may have been able to use `include` to add a private node module to the bundle before this change (by doing e.g. `include: ['./node_modules/some-dep/**']`). Even if you could have, I'm not sure if we'd want to support that, though. Thoughts?